frontendResourceBundles/src/main/resources/GitMacheteBundle.properties

```
@@ -250,6 +250,7 @@ string.GitMachete.EnhancedGraphTable.automatic-discover.success-message=Branch l
 string.GitMachete.EnhancedGraphTable.empty-table-text.cannot-discover-layout=Provided machete file ({0}) is empty and branch layout can''t be automatically detected
 string.GitMachete.EnhancedGraphTable.empty-table-text.loading=Loading...
 string.GitMachete.EnhancedGraphTable.empty-table-text.only-skipped-in-machete-file=None of the branches provided by machete file ({0}) exists within the local repository
+string.GitMachete.EnhancedGraphTable.duplicated-branches-text=The following branches defined by machete file appear more than once:
 string.GitMachete.EnhancedGraphTable.skipped-branches-text=The following branches defined by machete file do not belong to the local repository:
```

Btw, sounds more natural IMO:
```suggestion
string.GitMachete.EnhancedGraphTable.skipped-branches-text=The following branches listed in machete file do not belong to the local repository:
```

---

As in, technically it's git that _defines_ the branches, not us ;p we just reference them